### PR TITLE
Adds missing signatures (GLFW3 + emscripten C++ API)

### PR DESCRIPTION
**Pull request by gl84**
Adds signatures for GLFW3 and adds (some) for the the emscripten C++ API

### CODE REVIEW

File: src/library_glfw.js [hunk 1]
```diff
@@ -1201,35 +1208,42 @@ var LibraryGLFW = {
...
   glfwSwapInterval__deps: ['emscripten_set_main_loop_timing'],
+  glfwSwapInterval__sig: 'vi',
   glfwSwapInterval: function(interval) {
     interval = Math.abs(interval); // GLFW uses negative values to enable GLX_EXT_swap_control_tear, which we don't have, so just treat negative and positive the same.
     if (interval == 0) _emscripten_set_main_loop_timing(0/*EM_TIMING_SETTIMEOUT*/, 0);
     else _emscripten_set_main_loop_timing(1/*EM_TIMING_RAF*/, interval);
   },
 
 #if USE_GLFW == 3
+  glfwGetVersionString__sig: 'viii',
```
Comment (kripken) [inline src/library_glfw.js:1225]:
```suggestion
  glfwGetVersionString__sig: 'i',
```

**Review by kripken — APPROVED** [review]
lgtm with one that I think was wrong.

**Comment by bdenhollander** [comment]
@gl84 Thanks for adding `glfwSetErrorCallback__sig`. It would be great to get this merged!

**Comment by kripken** [comment]
Applied feedback and rebased. Merging.

**Pull request merged**